common/jmeter/saml/SAML2_SSO_Request_Path_Authentication.jmx

```diff
@@ -239,7 +246,7 @@ vars.put(&quot;sectoken&quot;, new String(encodeCredentials));
         <hashTree>
           <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="200 HTTP Code Response Assertion" enabled="true">
             <collectionProp name="Asserion.test_strings">
-              <stringProp name="-1606201635">HTTP/1.1 200</stringProp>
+              <stringProp name="103171775">HTTP/1.1 200</stringProp>
```

Comment:
why do we need this change

Reply:
Reverted by https://github.com/wso2/performance-is/pull/128/commits/ef88a709e62a267d93e2cf768f66851ed0a8035e